packages/fluentui/docs/src/polyfills.ts

```diff
@@ -0,0 +1,4 @@
+export const initPolyfills = () => {
```

Comment:
Out of curiosity, what's this?

Reply:
oh good catch; this should not be there.

Reply:
deleted